src/CompositeReferenceFieldManager.php

```diff
@@ -78,10 +89,58 @@ public function entityDelete(EntityInterface $entity, FieldDefinitionInterface $
     if (!$this->isCompositeField($field_definition)) {
       return;
     }
-    $referenced_entities = $entity->get($field_definition->getName())->referencedEntities();
```

Comment:
Instead of `@see composite_reference_entity_delete()` should be `@see composite_reference_entity_predelete()`

Reply:
Yes.